frappe/public/js/frappe/ui/dialog.js

```diff
@@ -232,6 +232,7 @@ frappe.ui.Dialog = class Dialog extends frappe.ui.FieldGroup {
 		this.get_minimize_btn().html(frappe.utils.icon(icon));
 		this.on_minimize_toggle && this.on_minimize_toggle(this.is_minimized);
 		this.header.find('.modal-title').toggleClass('cursor-pointer');
+		this.is_minimized ? $("body").css("overflow", "auto") : $("body").css("overflow", "hidden");
```

Comment:
- you can put the condition inside `.css()`
- I think some other CSS might be present to hide `overflow-x` which may break.